### ISSUE TITLE
Voiceover tells me there are 4 episodes in this list, but opening it shows them all as, “archived”

### ISSUE THREAD

**Issue by jldail13** [Automattic/pocket-casts-ios#4126]
### Description

The playlist feature is not working as expected.

### Step-by-step reproduction instructions

The playlist feature is not working as I’d expect.
If you enable Voiceover, create a playlist if you don’t already have one & add a few podcasts to it. For example, I have one for all my tech-related podcasts.
Either listen to or manually archive all episodes.
I’ve done this & when I put Voiceover focus on my April 10 playlist, it says, “4 episodes” b it when I open it, by double-tapping, it says all episodes have been archived, which should be true.
If that’s the case & I manually archived or listened to them, then why would it show 4 episodes, prior to opening it?

### Screenshots or screen recording

<img width="1376" height="1032" alt="Image" src="https://github.com/user-attachments/assets/16e3bd92-1723-415e-bae8-0416ef353b01" />
https://github.com/user-attachments/assets/e029b06e-86e6-4cdc-aad8-932758d6f7ee

### Did you search for existing bug reports?

- [x] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

IPadOs 26.5, iPad Pro 13-inch M4 & latest beta version of Pocket Casts

**Comment by SergioEstevao** [comment]
@MiSikora I confirmed that the above happens both on iOS and Android, is this expected behaviour or we should update the counts taking in account the archive status?

**Comment by MiSikora** [comment]
In the designs it is expected: 5sC4z4Mu42LvL4MAAIbQVi-fi-2960_28125

From my understanding showing 0 here would be misleading as users might think that they have more room in the playlist than available. But you'd have to ask the design team for the rationale.
